docs/guides/developer/docs/installation/source-linux.md

```diff
@@ -0,0 +1,120 @@
+Developer installation guide
+===========================
+
+These instructions outline how to install an all in one Opencast system on Linux.
+
+
+Get Opencast source:
+
+You can get the Opencast source code by either downloading a tarball of the source code or by cloning the Git
```

Comment:
It's for developers. hence, we should omit the tarball option. They should use git. Otherwise, they might end up having trouble creating pull requests, … That will also shorten this documentation quite a bit which–as a developer–you probably like ;-)